src/command/init/helpers.rs

```diff
@@ -26,7 +26,7 @@ pub fn display_project_created_message(
     println!("API Key: TODO");
 
     println!("\n️▲ Before you proceed:");
-    println!("- Set your graph API key as an environment variable (learn more about env vars by running `rover docs open config`)");
+    println!("- Set your graph API key as an environment variable (learn more about env vars by running `rover docs open configuring`)");
```

Comment:
We generally try to [avoid parentheses](https://developers.google.com/style/parentheses)

```suggestion
    println!("- Set your graph API key as an environment variable; learn more about env vars by running `rover docs open configuring`");
```